google-cloud-clients/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -157,9 +157,4 @@ public void testAggregatedListDiskTypes() {
       assertThat(diskType.getDefaultDiskSizeGb()).isNotNull();
     }
   }
-
-  /** For a given resource's URI, trim the path until it contains only the PathTemplate string. */
-  private static String trimUrl(String url) {
```

Comment:
We don't have to do this manually any more, since parsing a URL resource path will be built into the ResourceName.parse() function.